modules/rosa-sts-cluster-terraform-file-creation.adoc

```diff
@@ -70,7 +47,8 @@ resource "time_sleep" "wait_10_seconds" {
 EOF
 ----
 
-.. Create the `main.tf` file by running the following command:
+. Create the `main.tf` file by running the following command:
```

Comment:
Add some sort of conditionalized note for the customizations module along these lines:

[NOTE]
====
Copy and then edit this file *before* running the command to customize your <list whatever can be customized in this file>.
====

Reply:
I don't know why it is so big and in bold. Sorry. It looks like I am yelling.